libs/ui/inputs/src/lib/dropdown-multiselect/dropdown-multiselect.component.ts

```diff
@@ -74,9 +77,10 @@ export class DropdownMultiselectComponent {
   }
 
   get filteredChoicesByText() {
-    return this.choices.filter((choice) =>
-      choice.label.toLowerCase().includes(this.searchInputValue?.toLowerCase())
-    )
+    const filterPattern = this.searchInputValue?.toLowerCase()
```

Comment:
`?.toLowerCase()` might not be necessary anymore here, as the `FuzzyFilter` seems to handle this.

Reply:
right!